src/pagination.rs

```diff
@@ -262,12 +365,24 @@ fn interpret_permalink(
     config: &PaginationConfig,
     doc: &Document,
     index_num: usize,
+    index_title: Option<&liquid::value::Value>,
```

Comment:
This is still a major open question in the RFC issue

Reply:
If I got it correctly, we're going to use `index` for this, and `page_num` for the current page number in the current paginator